monarchmoney/monarchmoney.py

```diff
@@ -2377,6 +2457,71 @@ async def upload_account_balance_history(
             if resp.status != 200:
                 raise RequestFailedException(f"HTTP Code {resp.status}: {resp.reason}")
 
+            response = await resp.json()
+            session_key = response["session_key"]
+            return session_key
```

Comment:
return the entire JSON block, for consistency with the rest of the API.

Reply:
I refactor this quite a bit since the initial review so I've added `_upload_form_data` as a way to generalize the uploading form data (I'm hoping to use this later to bulk add transactions from a csv). Anyway, this should now return the whole response.